config/sync/og.og_role.rdf_entity-collection-author.yml

```diff
@@ -0,0 +1,34 @@
+uuid: ea4be6a5-b4cb-42c0-aee4-796e4776fd2e
+langcode: en
+status: true
+dependencies:
+  config:
+    - rdf_entity.rdfentity.collection
+id: rdf_entity-collection-author
+label: Member
+weight: 1
+is_admin: null
+group_type: rdf_entity
+group_bundle: collection
+group_id: null
+permissions:
+  - 'create discussion content'
+  - 'create document content'
+  - 'create event content'
+  - 'create news content'
+  - 'create solution rdf_entity'
+  - 'delete own comments'
+  - 'delete own custom_page content'
+  - 'delete own discussion content'
+  - 'delete own document content'
+  - 'delete own event content'
+  - 'delete own news content'
+  - 'delete own news content'
+  - 'edit own custom_page content'
+  - 'edit own discussion content'
+  - 'edit own document content'
+  - 'edit own event content'
+  - 'edit own news content'
+  - 'post comments'
+  - 'view own unpublished content'
```

Comment:
All of these permissions are already present in the standard member role, so they should not be duplicated here.